pumpkin/src/entity/player.rs

```diff
@@ -871,6 +873,9 @@ impl Player {
             SSwingArm::PACKET_ID => {
                 self.handle_swing_arm(SSwingArm::read(bytebuf)?).await;
             }
+            SUpdateSign::PACKET_ID => {
+                //TODO
```

Comment:
So this is empty, So Im predicting that Signs currently do not work right ?.
I recommend to make this a draft PR then when its still WIP

Reply:
In the game the sign updates with the text. I believe that this packet is suppose to update the server's nbt file--from what I have been reading. Couldn't test to see if another client sees the modified sign since I don't have another account.

Reply:
Figured it out, my bad. My latest commit implements the function that changes the sign for all clients by sending a client bound Block Entity Data packet.